diagnostics/model_accuracy/verify_inference_model.py

```diff
@@ -134,6 +105,12 @@ def check_accuracy(model, p, tolerance=0.001):
         help=
         'Model name to run inference. Availble models are Inception_v4, MobileNet_v1, ResNet50_v1',
         required=True)
+
+    parser.add_argument(
+        '--json_file_name',
+         help=
+         'json file to use for run inference and check acucuracy',
```

Comment:
Maybe we could modify --help here to `json file with model parameters to run inference and accuracy values to verify`

Reply:
Thanks @sindhu-nervana, good catch, I will change it.